### ISSUE TITLE
Caching / new releases

### ISSUE THREAD

**Issue by adamunchained** [soonaverse/app#135]
There are issues with angular caching. This causes an issue for users when new release of the application goes out and users are getting mixture of new javascript files and old ones. (JS files generated by Angular compilation process). This can be very frustrating for users that didn't navigate to the application for some time.

This results in broken application or certain things not loading properly.

Todo:

- [ ] debug and investigate how caching works and how it can be improved
- [ ] implement flushing of CloudFlare cache?
- [ ] disable caching on client browser or improve it

**Comment by CptBananu** [comment]
Hi @adamunchained ,

not 100% sure if my **minor** issue fits here but I recently recognized a correlation of the staked tokens in my profiles activity tab to the last stakable token displayed in Soonaverse:

First I was on the EpochZero-Space checking my staked FUEL balance on the right. After that I went to my profile and it showed my staked FUEL amount there instead of staked SOON. Also the staked value below my red mark is calculated "correctly".

I didn't think that this deserves a new github issue bc the effect on users is negligible and can easily be fixed on client side by simply refreshing the page (no cache clear needed). That's why I checked the issues and added it here. But ofc I can create a dedicated issue for this if u like.

![image](https://github.com/soonaverse/app/assets/104099536/66a96f44-18de-4568-b44c-30cd594e6eae)

Greetings,
Bananu
